notifier/src/test/kotlin/modules/JiraNotifierTest.kt

```diff
@@ -216,6 +217,34 @@ class JiraNotifierTest : WordSpec({
         }
     }
 
+    "changeAssignee" should {
+        "succeed for valid input" {
+            val issueClient = mockk<IssueRestClient>()
```

Comment:
Consider factoring out the redundant common setup in lines 222 - 225.

Reply:
Done: https://github.com/oss-review-toolkit/ort/pull/5179